src/pages/[platform]/build-a-backend/add-aws-services/logging/set-up-logging/index.mdx

```diff
@@ -0,0 +1,486 @@
+import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
+
+export const meta = {
+  title: 'Set up Logging',
+  description: 'Learn how to set up Amplify logging',
+  platforms: [
+    'swift',
+    'android'
+  ]
+};
+
+export const getStaticPaths = async () => {
+  return getCustomStaticPath(meta.platforms);
+};
+
+export function getStaticProps(context) {
+  return {
+    props: {
+      meta
+    }
+  };
+}
+
+The Amplify Logger enables you to troubleshoot and debug issues with your apps, to help you provide the best experience for your customers. You can log messages for errors by the Amplify library and add custom logs as well and send them to Amazon CloudWatch. With the Amplify Logger, you also can remotely change your logging configuration to adjust your logging levels, or add an allow list of customer IDs to help you detect issues more granularly for your apps in production.
+
+## Prerequisites
+
+<InlineFilter filters={['android']}>
+
+
+
+
+* An Android application targeting at least Android SDK API level 24 with Amplify libraries integrated
+    * For a full example, please follow the [mobile support walkthrough](/swift/start/mobile-support/) .
```

Comment:
Should this link to the `android` docs since it's in the android inline filter?
```suggestion
    * For a full example, please follow the [mobile support walkthrough](/android/start/mobile-support/) .
```